app/lib/features/chat/widgets/bubble_builder.dart

```diff
@@ -58,38 +58,17 @@ class BubbleBuilder extends ConsumerWidget {
         isMemberEvent
             ? child
             : SwipeTo(
-                onLeftSwipe: !isAuthor
-                    ? null
-                    : (DragUpdateDetails details) {
-                        FocusScope.of(context)
-                            .requestFocus(chatInputFocusState.state);
-                        if (chatInputState.currentMessageId != null) {
-                          chatInputNotifier.emojiRowVisible(false);
-                          chatInputNotifier.setRepliedToMessage(message);
-                          chatInputNotifier.showReplyView(true);
-                          chatInputNotifier.setReplyWidget(child);
-                        } else {
-                          chatInputNotifier.showReplyView(true);
-                          chatInputNotifier.setRepliedToMessage(message);
-                          chatInputNotifier.setReplyWidget(child);
-                        }
-                      },
+                onLeftSwipe: (DragUpdateDetails details) {
+                  FocusScope.of(context)
+                      .requestFocus(chatInputFocusState.state);
+                  chatInputNotifier.setReplyToMessage(message);
```

Comment:
What about `isAuthor` condition here?

Reply:
why? you can reply to your own message ...